pyInterface/package/_fileManager.py

```diff
@@ -304,102 +289,100 @@ def _getAmplitudeFilePaths(self):
-		for binID in self.getBinIDList():
+		integralFiles = collections.OrderedDict()
+		for binID, _ in enumerate(self.binList):
 			for eventsType in [EventsType.GENERATED, EventsType.ACCEPTED]:
 				integralFiles[(binID, eventsType)] = self.integralDirectory + "/integral_binID-" + str(binID) + "_" + str(eventsType) + ".root"
 		return integralFiles
 
 
 	def _openDataFiles(self):
-		dataFileNames = glob.glob(self.dataDirectory + "/*.root")
+		dataFileNames = sorted(glob.glob(self.dataDirectory + "/*.root"))
 		inputFiles = {}
-
 		for dataFileName in dataFileNames:
 			dataFile = ROOT.TFile.Open(dataFileName, "READ")
 			if not dataFile:
 				pyRootPwa.utils.printErr("could not open event file '" + dataFileName + "'.")
-				return {}
+				return collections.OrderedDict()
 			eventMeta = pyRootPwa.core.eventMetadata.readEventFile(dataFile)
 			if not eventMeta:
 				pyRootPwa.utils.printErr("could not find metadata in event file '" + dataFileName + "'.")
-				return {}
-			inputFile = InputFile(dataFileName, eventMeta.binningMap(), fileManager.eventsTypeFromBpEnum(eventMeta.eventsType()))
+				return  collections.OrderedDict()
+			inputFile = InputFile(dataFileName,
+			                      eventMeta.binningMap(),
+			                      fileManager.pyEventsType(eventMeta.eventsType()),
+			                      eventMeta.additionalSavedVariableLables())
 			if inputFile.eventsType not in inputFiles:
 				inputFiles[inputFile.eventsType] = []
 			inputFiles[inputFile.eventsType].append(inputFile)
 			dataFile.Close()
-
-		if not inputFiles:
-			pyRootPwa.utils.printErr("no binning maps found.")
-			return {}
-		for eventsType in inputFiles:
-			if not inputFiles[eventsType]:
-				pyRootPwa.utils.printErr("no binning maps found for eventsType '" + eventsType + "'.")
-				return {}
-
+		retval =  collections.OrderedDict()
+		for eventsType in sorted(inputFiles.keys()):
+			retval[eventsType] = inputFiles[eventsType]
 		return inputFiles
 
 
 	def _openKeyFiles(self):
 		keyFileNames = glob.glob(self.keyDirectory + "/*.key")
 		keyFiles = {}
 
-		for keyFileID in range(len(keyFileNames)):
-			keyFileName = keyFileNames[keyFileID]
+		for keyFileName in keyFileNames:
 			waveDescriptions = pyRootPwa.core.waveDescription.parseKeyFile(keyFileName)
-			if len(waveDescriptions) == 0:
+			if not waveDescriptions:
 				pyRootPwa.utils.printErr("could not read wave description from key file '" + keyFileName + "'.")
-				return []
-			for waveDescriptionID in xrange(len(waveDescriptions)):
-				waveDescription = waveDescriptions[waveDescriptionID]
+				return collections.OrderedDict()
+			for waveDescriptionID, waveDescription in enumerate(waveDescriptions):
 				(success, amplitude) = waveDescription.constructAmplitude()
 				if not success:
 					pyRootPwa.utils.printErr("could not construct decay topology for wave descrption at index " + str(waveDescriptionID) + " of key file '" + keyFileName + "'.")
-					return []
+					return collections.OrderedDict()
```

Comment:
the usual return value seems to be a standard dictionary, vie return an `OrderedDict` in case of error?